lib/parsers/rw2_parser.rb

```diff
@@ -19,8 +30,13 @@ def call(io)
 
     # RW2 doesn't use the standard EXIF width and height tags (🤷🏻). We can compute them from the sensor
     # top/bottom/left/right border tags. See https://exiftool.org/TagNames/PanasonicRaw.html for more.
-    w = sensor_border(exif, :right) - sensor_border(exif, :left)
-    h = sensor_border(exif, :bottom) - sensor_border(exif, :top)
+    left_sensor_border = sensor_border(exif, :left)
+    right_sensor_border = sensor_border(exif, :right)
+    w = left_sensor_border && right_sensor_border && right_sensor_border - left_sensor_border
```

Comment:
I find this a bit confusing; should it return a boolean? Is it `nil` or `false` when the conditions are not met? `&&` and `||` are traditionally used when after a boolean value, so I would write these in a more _expected_ way.